src/frontend/AppContainer.tsx

```diff
@@ -1,5 +1,8 @@
 import { createAppContainer } from "react-navigation";
 
+import { AppStack } from "./NavigationStacks/AppStack";
 import { WithModalsStack } from "./NavigationStacks/WithModalsStack";
 
-export default createAppContainer(WithModalsStack);
+export default createAppContainer(
+  process.env.FEATURE_ONBOARDING ? WithModalsStack : AppStack
```

Comment:
Generally, are we okay with just checking the existence of the flag as opposed to a specific value i.e. not checking if `process.env.FEATURE_ONBOARDING === "true"`

Reply:
think i'm gonna opt for making it more explicit so that we check for the actual value too